.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,8 @@
 blank_issues_enabled: false
 contact_links:
-  - name: Official Discord server
+  - name: Discord Support
     url: https://discord.premid.app
-    about: Please use our ticket system for urgent requests!
+    about: If you are experiencing issues with your PreMiD, go to our Discord server for help.
+  - name: Feature Requests & Presence Requests
+    url: https://github.com/PreMiD/Presences/discussions
```

Comment:
```suggestion
    url: https://github.com/PreMiD/Presences/discussions/new
```

Reply:
Nope, as they need to copy the template before opening the discussion

Reply:
Ah right, good luck with getting people to notice that
Might as well just link that discussion and have a link to the discussion creation page over there